.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: ['react', '@typescript-eslint'],
+  rules: {
+    indent: ['error', 2],
+    semi: ['error', 'always'],
+    'import/prefer-default-export': 'off',
+    'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-filename-extension': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'import/no-unresolved': 'off',
+    'import/extensions': 'off',
+    'explicit-function-return-type': 'off',
+    'arrow-body-style': 'off',
```

Comment:
```suggestion
    'arrow-body-style': ["error", "as-needed"],
```
완전히 풀기보다는, 일관성이 지켜지도록 as-needed 는 어떤지!

https://eslint.org/docs/rules/arrow-body-style#as-needed